internal/lib/sandbox/namespaces_linux.go

```diff
@@ -10,8 +10,8 @@ import (
 	"sync"
 
 	nspkg "github.com/containernetworking/plugins/pkg/ns"
+	"github.com/containers/storage/pkg/mount"
```

Comment:
ah you know what, maybe it's the storage mount package working differently, specifically with the umount below

Reply:
Alright this makes sense. I’ll check that out tomorrow 👍

Reply:
Working on a fix: https://github.com/containers/storage/pull/561